utils/CircuitCheck/UnitaryBuilder.cpp

```diff
@@ -234,7 +234,7 @@ void UnitaryBuilder::growMatrix(unsigned numQubits) {
 //     q : ─┤    ├┤ U ├─   ─┤    ├┤ U ├─   ─┤    ├┤     ├─   ─┤    ├─
 //      0   │    │└───┘     │    │└───┘     │    ││     │     │    │
 //          │    │          │    │┌───┐     │    ││     │     │    │
-//     q : ─┤ C  ├────── = ─┤ C  ├┤   ├─ = ─┤ C  ├┤ I⊗U ├─ = ─┤ C  ├─
+//     q : ─┤ C  ├────── = ─┤ C  ├┤   ├─ = ─┤ C  ├┤ I⊗U├─ = ─┤ C  ├─
```

Comment:
I don't think we wan to delete this space.

Reply:
Updated in c844c7fd